page-sidebar.php

```diff
@@ -18,49 +18,22 @@
 
 
 get_header(); ?>
+<?php get_template_part('inc/sub-headerException'); ?>
+		
+		<div id="stage" class="inner" role="main">
 
-		<?php get_template_part('inc/breadcrumbs', 'noChild'); ?>
+			<?php while ( have_posts() ) : the_post(); ?>
+
+				<div id="content" class="content has-sidebar">
+
+				<?php get_template_part( 'inc/content', 'page' ); ?>
```

Comment:
The issue seems to be that line 29 should call `get_template_part( 'content', 'page' );` rather than `get_template_part( 'inc/content', 'page' );`? This is a reference to content-page.php from the Parent theme, but that template is in the root directory, rather than the `inc/` directory.